.gitattributes

```diff
@@ -0,0 +1,38 @@
+# A-Frame .gitattributes
+# Directories and files to be ignored from release archives
+# keep it consistent with .npmignore
+
+examples export-ignore
```

Comment:
I think we should keep these for now, just in case anyone wants them for whatever reason. And then we can merge the rest.